### PR TITLE
Rewrite legal docs

### PR DESCRIPTION
**Pull request by vednoc**
WIP. Looking forward to feedback once the rewrite of Privacy Policy is done.

### CODE REVIEW

File: docs/privacy-policy.md [hunk 1]
```diff
@@ -1,178 +1,157 @@
...
 
-By visiting the relative links stated above, we save a unique hash combined of your IP address, the style ID, reffered within the relative link as `:id`, and a secret key. This unique hash is not easily reversible without brute-forcing all public IP addresses in IPv4 address space and our secret key. This provides decently accurate style statistics while still respecting your privacy. 
+## How do we store your data?
 
-## HOW DO WE USE YOUR INFORMATION? 
+We follow best practices when it comes to handling your data and store
+everything on our own server.
 
-_In Short: We process your information for purposes based on legitimate business interests, the fulfillment of our contract with you, compliance with our legal obligations, and/or your consent._  
+The data used for userstyle statistics is stored as a unique hash, and is not
```
Comment (Gusted) [inline docs/privacy-policy.md:80]: For some reason in this sentence I feel like the need of "anonymizing the IP trough an hash that's combined with a secret key"

File: docs/privacy-policy.md [hunk 2]
```diff
@@ -1,178 +1,157 @@
...
+Yes, we use cookies for the purpose of keeping you signed in and authorizing
+various actions on the Site that require an account.
 
-_In Short: No, not to our knowledge._
 
-Our server is provided by vultr.com, we recommend that you review their privacy policy to understand how they collect, use and share your personal information. Our server is located in Amsterdam, Netherlands. All traffic from userstyles.world is processed by this server.
+## Do we use any external cookies?
 
-If you aren’t a resident in the European Economic Area, then these countries may not have data protection or other laws as comprehensive as those in your country. We will however take all necessary measures to protect your personal information in accordance with this privacy policy and applicable law. 
+No. And we never will.
```
Comment (Gusted) [inline docs/privacy-policy.md:134]: "never" you're promising something.

File: docs/privacy-policy.md [hunk 3]
```diff
@@ -1,178 +1,157 @@
...
+any related services. Please read it carefully as it will help you make informed
+decisions about sharing your personal information with us.
 
-## WHAT INFORMATION DO WE STORE? 
 
-### **Personal information you disclose to us**
+## What data do we collect?
 
-_In Short: We store personal information that you provide to us such as email address._
+We collect the following data:
```
Comment (Gusted) [inline docs/privacy-policy.md:49]: We actually have more data if you provide them to use e.g. the social media thingies.

Comment (Gusted) [reply docs/privacy-policy.md:49]: Maybe make it more general but still have the Username/Email Address. Account details (ex Username, email)

**Review by Gusted — COMMENTED** [review]
Yep exactly as I had it in mind :D. Good hob.

**Comment by the-j0k3r** [comment]
1) Get someone with actual legal practical experience to have a look atboth your terms of service and privacy policy. e.g. a law teacher or an actual lawyer, some European countries have Advice Bureaus that provide legal services for free or at a very small fee.

Don't presume you know better than an actual lawyer. The current live documents are deeply flawed in key areas.

Legal terminology is not up for improvements/correction by anyone who has no such legal experience. No matter your proficiency of the English language and your interpretation of the language used or assumptions that no one reads these documents.

It is ill advised to copy and paste some 3rd part website/service and modify that to suit your needs, when the ppl doing the modifications dont understand that these things once uploaded and live, **form a legal and binding contract between operators/users.** This contract is not imaginary, this is real.

Avoid conflicts between clauses that places anyone involved/operators at risk of legal action (however unlikely that scenario is even for something as simple as userstyles) such as assuming control of copyright or submitted content and or intellectual property or implying any marketing usage that may or may not result in any profit, even if such profit covers operating costs.
If there are any interests about any of this, at least be sure your asses are legally covered.

TLDR
USW is a great service to the community of developers and users, we have a great example of how not to handle this service i.e. ala USo, its birth should not be put at risk by any risk of legal action to its operators, however unlikely that may seem or maybe or may not be.

As is, and in good conscience, I cannot be involved in any capacity, be it contributing to the service as a developer of userstyles or user, or anything associated with the service, beyond this comment.

**Comment by vednoc** [comment]
You're making a comment based off on what we had previously. Both documents are going away.

The intent behind the new "privacy policy" is to explain the ways in which we handle the data because not everyone is going to read the code and figure it all out on their own. Perhaps the name should be changed to something else.

We're simply trying to be open and transparent in order to gain trust. Using simple, plain English, is going to encourage more users to read and understand our practices. If something isn't clear, we're always going to address it in the best way possible.

In the end, I can only know so much. This is where others have to chime in, otherwise mistakes are inevitable.

**Comment by the-j0k3r** [comment]
Only a lawyer is qualified to write/rewrite these, not you or anyone else who isn't qualified, by attempting this, you are open to liability.

**Comment by vednoc** [comment]
There are no "these" here, they have been removed.

Open to liability? I *am* liable for all the data that is on the server and what happens to it.

The rewrite of our privacy policy — or whatever you want to call it because at this point it should be renamed — is purely our good will to keep our users informed about our practices. We don't need to have hundreds of lines worth of lawyer-level English. Does that make any sense?

File: docs/privacy-policy.md (hunk 1 above)
Comment (vednoc) [reply docs/privacy-policy.md:80]: That is exactly what I was thinking of.

File: docs/privacy-policy.md (hunk 2 above)
Comment (vednoc) [reply docs/privacy-policy.md:134]: Yeah, but are we ever going to use cookies from third parties?

File: docs/privacy-policy.md (hunk 3 above)
Comment (vednoc) [reply docs/privacy-policy.md:49]: Right, I'll revise it.

File: docs/privacy-policy.md (hunk 2 above)
Comment (Gusted) [reply docs/privacy-policy.md:134]: Not _likely_, you can't predict the future. And if you now say never will and later on we for some absurd reason needs to it's very conflicting feelings.

File: docs/privacy.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+We use your data in order to show userstyle statistics.
+
+
+## How do we store your data?
+
+We follow best practices when it comes to handling your data and store
+everything on our own server.
+
+The data used for userstyle statistics is anonymizing the IP address through a
```
Comment (Gusted) [inline docs/privacy.md:87]: Read this sentence up until hash function. It reads smooth rif you have the hash function... Anomymzed IP Adress etc

File: docs/privacy.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+## How do we use your data?
+
+We use your data in order to show userstyle statistics.
+
+
+## How do we store your data?
+
+We follow best practices when it comes to handling your data and store
+everything on our own server.
```
Comment (Gusted) [inline docs/privacy.md:85]:
```suggestion
everything on our server.
```
Redundancy. our & own is referring to the same definition.

File: docs/privacy.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+The data used for userstyle statistics is anonymized by using a hash function
+with a secret key. It is not easily reversible without brute-forcing all public
+IP addresses in IPv4 address space in combination with said secret key. This
```
Comment (Gusted) [inline docs/privacy.md:89]:
```suggestion
IP addresses in IPv4 address space in combination with the said secret key. This
```
We've already mentioned secret key.

File: docs/privacy.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+Try it out online:
+
+- [HMAC hash generator](https://cryptii.com/pipes/hmac)
+- [Convert String to Bytes](https://onlinestringtools.com/convert-string-to-bytes)
+
+
+## What are your data protection rights?
+
+- The right to be informed – You have the right to know what personal data is
+  collected, why, how long it will be kept, how you can file a compmlaint, and
```
Comment (Gusted) [inline docs/privacy.md:117]:
```suggestion
  collected, why, how long it will be kept, how you can file a complaint, and
```

File: docs/privacy.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+information doesn't reveal your identity. It comes from default NGINX server
+logs, which we flush every 24 hours, and includes the following data:
+
+```
+# IP        Date                         Visited page                  Browser Agent
+1.2.3.4 - - [06/Jun/2021:23:06:13 +0000] "GET / HTTP/1.1" 200 5217 "-" "Mozilla/5.0 ..."
+```
+
+This information is necessary for maintaining the security and operation of
+our Site, and for our internal analytic and reporting purposes.
```
Comment (Gusted) [inline docs/privacy.md:149]:
```suggestion
our Site, and for our internal analytics and reporting purposes.
```

File: docs/privacy.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+
+## Do we use cookies?
+
+Yes, we use cookies for the purpose of keeping you signed in and authorizing
+various actions on the Site that require an account.
+
+
+## Do we use any third party cookies?
```
Comment (Gusted) [inline docs/privacy.md:158]:
```suggestion
## Do we use any third-party cookies?
```

**Review by Gusted — APPROVED** [review]
Just grammar this time. The content itself was good from my perspective.

**Comment by vednoc** [comment]
Should be good to go now. A public reminder that I need to set up spell checking in Emacs. 😄
